docs/design/coreclr/jit/lsra-detail.md

```diff
@@ -321,7 +323,12 @@ After LSRA, the graph has the following properties:
 
     -   Note that a write-thru variable def is always written to the stack, and the `GTF_SPILLED`
         flag (not otherwise used for pure defs) to indicate that it also remains live
-        in the assigned register.
+        in the assigned register. This is somewhat counter-intuitive, but
```

Comment:
May be we should move this after we describe what a node with `GTF_SPILLED`  means and further re-phrase as :

"If a node has both `GTF_SPILL` and `GTF_SPILLED`, the tree node is spilled after it is evaluated (`GTF_SPILL`) and reloaded prior to the use (`GTF_SPILLED`). After the node is spilled and before the reload, its value is *not* live in the register. However, for EH-write-thru variables, there is an exception. The variable def is *always* written to the stack (`GTF_SPILL`), should always be reloaded (`GTF_SPILLED`) and *always* remain live in the register as well."

Reply:
Sounds good, though the description should be reordered to make it clear that the reloading comes first in the general case.